ethcore/src/header.rs

```diff
@@ -219,8 +219,8 @@ impl Header {
 		s.append(&self.timestamp);
 		s.append(&self.extra_data);
 		if let Seal::With = with_seal {
-			for b in &self.seal { 
-				s.append_raw(&b, 1); 
+			for b in &self.seal {
```

Comment:
`self.seal` is a vector of post-RLP-encoded fields. It is *not* pre-RLP-encoded data. The reason for this is to allow for complex fields as part of the seal without having meta-RLP within the header.